app/services/contact_establishment_data_migration.rb

```diff
@@ -0,0 +1,19 @@
+# Data Migration service
+# This service is designed to be used once to update the category for all Contact::Establishment records
+#
+# Usage:
+#
+# `bin/rails runner "ContactEstablishmentDataMigration.new.migrate!"`
+#
+class ContactEstablishmentDataMigration
+  def migrate!
+    contacts = Contact::Establishment.all
+    puts "#{contacts.count} Contact::Establishment records to update"
+
+    contacts.each do |contact|
```

Comment:
You could just `update_all`: https://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-update_all